cpp/include/cuvs/neighbors/cagra.hpp

```diff
@@ -229,6 +229,13 @@ struct search_params : cuvs::neighbors::search_params {
    * impact on the throughput.
    */
   float persistent_device_usage = 1.0;
+
+  /**
+   * A parameter indicating the rate of nodes to be filtered-out, when filtering is used.
+   * The value must be equal to or greater than 0.0 and less than 1.0. Default value is
+   * negative, in which case the filtering rate is automatically set.
+   */
+  float filtering_rate = -1.0;
```

Comment:
This is interesting. So is specifying this value an alternative to explicitly computing the sparsity of the pre-filtering bitset (which would introduce additional compute into the search)?

Reply:
It would be clearer to write "automatically computed" rather than "automatically set" here. I will fix it.

Reply:
I like that. "automatically compted" is perfect.